test/rtf-test-files/colors/expected.html

```diff
@@ -0,0 +1,16 @@
+<div style="min-height: 8pt; text-align: left;">
```

Comment:
I assume your editor formatted this file. That breaks the test though. When you regenerate the test case this should be fixed automatically.